CODEOWNERS

```diff
@@ -0,0 +1,14 @@
+*  @fjetter
```

Comment:
I added this because it reflects reality. But @fjetter if you would prefer not to add this line we can remove it.

Reply:
👍